utils/zerovec/src/varzerovec/mod.rs

```diff
@@ -167,6 +167,30 @@ impl<'a, T: AsVarULE> VarZeroVec<'a, T> {
             VarZeroVecInner::Borrowed(components) => components.len(),
         }
     }
+
+    /// Returns `true` if the vector contains no elements.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use std::str::Utf8Error;
+    /// # use zerovec::VarZeroVecError;
+    /// # use zerovec::VarZeroVec;
+    ///
+    /// let strings: Vec<String> = vec![];
+    /// let bytes = VarZeroVec::get_serializable_bytes(&strings).unwrap();
+    ///
+    /// let mut vec: VarZeroVec<String> = VarZeroVec::try_from_bytes(&bytes)?;
+    /// assert!(vec.is_empty());
+    /// # Ok::<(), VarZeroVecError<Utf8Error>>(())
+    /// ```
+    pub fn is_empty(&self) -> bool {
+        match self.0 {
+            VarZeroVecInner::Owned(ref vec) => vec.is_empty(),
+            VarZeroVecInner::Borrowed(components) => components.is_empty(),
+        }
+    }
```

Comment:
That's the only functional extension - clippy flagged use of `.len() == 0` in several places and indicated that `is_empty` leads to better assembly and inlining, so I added it. Hope that's ok!